pages/room/index.js

```diff
@@ -27,9 +29,21 @@ const CreateRoom = ({ spotifyAuthURL, spotifyCode }) => {
     const session = await getSession();
     const { name, email } = session.user;
     const result = await axios.get('/api/user/get-user', { params: { name, email } });
+    const GET_USERS = gql`
+      query {
+        users(name: name, email: email) {
+          name
+          email
+          image
+        }
+      }
+    `;
+    const { loading, error, data } = useQuery(GET_USERS);
+    console.log('data', data);
```

Comment:
now that I think about it, why are we even doing this client side? We should just be getting the user data server side in getServerSideProps